ext/sqlite3/sqlite3.c

```diff
@@ -287,6 +287,51 @@ PHP_METHOD(sqlite3, lastErrorCode)
 }
 /* }}} */
 
+/* {{{ proto int SQLite3::lastExtendedErrorCode()
+   Returns the numeric extnded result code of the most recent failed sqlite API call for the database connection. */
+PHP_METHOD(sqlite3, lastExtendedErrorCode)
```

Comment:
This method needs to be registered in the `php_sqlite3_class_methods` struct and arginfo to make it available to the `SQLite3` class